src/main/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtil.java

```diff
@@ -417,4 +417,40 @@ else if (sibling.getType() == TokenTypes.SINGLE_LINE_COMMENT) {
                 || BlockCommentPosition.isOnPackage(blockComment));
     }
 
+    public static ArrayList<String> getContentOfSummaryTag(DetailNode javadoc) {
+        ArrayList<String> summaryTagContent = null;
+        for (DetailNode node : javadoc.getChildren()) {
+            if (node.getType() == JavadocTokenTypes.CUSTOM_NAME
+                    && node.getText().equals("@summary")
```

Comment:
Please remove the check for `@summary` text, we want this util method to be universal for all `CUSTOM_NAME` tags.

Reply:
Checkout my comment on the issue once, I want to make sure I have implemented what is required.

Reply:
done

Reply:
Resolved.

Reply:
Please see updated https://github.com/checkstyle/checkstyle/issues/9993#issuecomment-845768722 and resolve this review from your side.